build.gradle

```diff
@@ -5,7 +5,7 @@ apply plugin: 'spring-boot'
 apply plugin: 'groovy'
 apply plugin: 'signing'
 
-version = '2.0.1'
+version = '0.7.1.2'
 group = 'com.bericotech'
```

Comment:
@shlomisut - Sonatype requires us to have a unique GroupID, generally for a domain that we own (unless as John points out, the GroupID can't possibly be a valid internet domain).